modules/core/src/main/java/org/apache/ignite/IgniteCluster.java

```diff
@@ -579,4 +580,20 @@
      * @return Status of baseline auto-adjust.
      */
     public BaselineAutoAdjustStatus baselineAutoAdjustStatus();
+
+    /**
+     * Return a policy of shutdown or default walue {@link IgniteConfiguration.DFLT_SHUTDOWN_POLICY}
+     * if property did not set.
+     *
+     * @return Shutdown policy.
+     */
+    public ShutdownPolicy shutdownPolicy();
+
+    /**
+     * Set a shutdown policy on cluster.
```

Comment:
Here and below:
Set -> Sets,
on cluster -> on the cluster or on a cluster depending on what is meant.